packages/visualizations/addon/components/number-format-dropdown.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright 2018, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ *
+ * Usage:
+ *  {{number-format-dropdown
+ *    column=column
+ *    onUpdateFormat = (action 'onUpdateFormat')
+ *  }}
+ */
+
+import Component from '@ember/component';
+import layout from '../templates/components/number-format-dropdown';
+import { assign } from '@ember/polyfills';
+import { getWithDefault, set } from '@ember/object';
+
+export default Component.extend({
+  layout,
+
+  /**
+   * @property {Array} classNames
+   */
+  classNames: ['number-format-dropdown'],
+
```

Comment:
Can we add a property 
```
import { readOnly } from '@ember/computed';
format: readOnly('column.format'),
```

that way we won't need the `(or format column.format)` in the hbs. Clarifies things a little.

Reply:
format is current used as a placeholder for attempting format change, so this will break the implementation

Reply:
How would it break? 
`format` is either `column.format` which the readOnly would solve, or when a format is set by the `number-format-selector`, which sets `format` when something changes.

Reply:
![image](https://user-images.githubusercontent.com/5875315/40742448-35571a6a-6414-11e8-93b3-8a7484195c59.png)

If we make it readonly,  we will not be able to set it in `onUpdateFormat `

Reply:
Ok. we can use `oneWay` instead. https://www.emberjs.com/api/ember/3.0/functions/@ember%2Fobject%2Fcomputed/oneWay

Reply:
updated